rpc_jobs/rpc_designate.yml

```diff
@@ -41,10 +41,17 @@
       - xenial:
```

Comment:
What happens when `image` is `xenial` and `scenario` is `newton`, which `IMAGE` wins?

Reply:
Good point -- think I'll just create a new project for these snapshot images instead.